### PR TITLE
feat-android-getting-started

### PR DESCRIPTION
**Pull request by lohanidamodar**

### CODE REVIEW

File: app/views/docs/getting-started-for-android.phtml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+<div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+                    <pre class="line-numbers"><code class="prism language-kotlin" data-prism>// Register User
+val account = Account(client)
+val response = account.create("email@example.com", "password")
+val json = response.body?.string()
+</code></pre>
+</div>
+
+<h2>Full Example</h2>
+<div class="ide" data-lang="dart" data-lang-label="Dart SDK">
```
Comment (christyjacob4) [inline app/views/docs/getting-started-for-android.phtml:108]: You might wanna replace references to dart here

File: app/views/docs/getting-started-for-android.phtml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+<p>From the options, choose to add a new <b>Android</b> platform and add add your app <u>name</u> and <u>package name</u>, Your package name is generally the applicationId in your app-level build.gradle file. By registering your new app platform, you are allowing your app to communicate with the Appwrite API.</p>
+
+<h2>Get Appwrite Android SDK</h2>
+
+<p>First, add this to your root level build.gradle file: </p>
+<div class="ide" data-lang="groovy" data-lang-label="Groovy">
+  <pre class="line-numbers">
+  <code class="prism language-groovy" data-prism><?php echo $this->escape('repositories {
+    maven {
```
Comment (christyjacob4) [inline app/views/docs/getting-started-for-android.phtml:36]: This appears to be the configuration to publish to maven. Not to fetch the dependency.

You will have to add `mavenCentral()` to the `repositories` section under `buildscript`

Then include the dependency 

`implementation 'io.appwrite.sdk-for-android:0.0.1'`

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Some minor comments, will need to revisit this once its deployed

File: app/views/docs/getting-started-for-android.phtml (hunk 2 above)
Comment (lohanidamodar) [reply app/views/docs/getting-started-for-android.phtml:36]: Done!

File: app/views/docs/getting-started-for-android.phtml (hunk 1 above)
Comment (lohanidamodar) [reply app/views/docs/getting-started-for-android.phtml:108]: done

**Review by christyjacob4 — APPROVED** [review]
Looks Good :rocket:

File: app/views/docs/getting-started-for-android.phtml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+}');?>
+  </code>
+  </pre>
+</div>
+
+<p>And add this to your project's build.gradle file:</p>
+
+<div class="ide" data-lang="groovy" data-lang-label="Groovy">
+  <pre class="line-numbers">
+  <code class="prism language-groovy" data-prism>'implementation("io.appwrite.sdk-for-android:<?php echo $this->escape($androidVersion); ?>")'
```
Comment (abnegate) [inline app/views/docs/getting-started-for-android.phtml:46]:
```suggestion
  <code class="prism language-groovy" data-prism>'implementation("io.appwrite:sdk-for-android:<?php echo $this->escape($androidVersion); ?>")'
```

File: app/views/docs/getting-started-for-android.phtml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    $version = $lang['version'] ?? '';
+
+    if(!empty($key)) {
+        $versions[$key] = $version;
+    }
+}
+
+$androidVersion = (isset($versions['android'])) ? $versions['android'] : '';
+?>
+<p>Appwrite is a development platform providing you easy yet powerful API and management console to get your next project up and running quickly.</p>
```
Comment (abnegate) [inline app/views/docs/getting-started-for-android.phtml:16]:
```suggestion
<p>Appwrite is a development platform providing you an easy yet powerful API and management console to get your next project up and running quickly.</p>
```

File: app/views/docs/getting-started-for-android.phtml [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  <pre class="line-numbers">
+  <code class="prism language-groovy" data-prism>'implementation("io.appwrite.sdk-for-android:<?php echo $this->escape($androidVersion); ?>")'
+  </code>
+  </pre>
+</div>
+
+
+<h3>OAuth Callback</h3>
+
+<p>In order to capture the Appwrite OAuth callback url, the following activity needs to be added to your <a href="https://github.com/appwrite/playground-for-android/blob/app/src/main/AndroidManifest.xml" target="_blank" rel="noopener">AndroidManifest.xml</a>. Be sure to relpace the <b>[PROJECT_ID]</b> string with your actual Appwrite project ID. You can find your Appwrite project ID in you project settings screen in your Appwrite console.</p>
```
Comment (abnegate) [inline app/views/docs/getting-started-for-android.phtml:54]:
```suggestion
<p>In order to capture the Appwrite OAuth callback url, the following activity needs to be added to your <a href="https://github.com/appwrite/playground-for-android/blob/app/src/main/AndroidManifest.xml" target="_blank" rel="noopener">AndroidManifest.xml</a>. Be sure to replace the <b>[PROJECT_ID]</b> string with your actual Appwrite project ID. You can find your Appwrite project ID in you project settings screen in your Appwrite console.</p>
```

File: app/views/docs/getting-started-for-android.phtml [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+<div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+    <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import io.appwrite.Client
+import io.appwrite.services.Account
+
+val client = Client(context)
+  .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+  .setProject("5df5acd0d48c2") // Your project ID
+  .setKey("919c2d18fb5d4...a2ae413da83346ad2") // Your secret API key
+  .setSelfSigned() // For self signed certificates only use for development
```
Comment (abnegate) [inline app/views/docs/getting-started-for-android.phtml:81]:
```suggestion
  .setSelfSigned(true) // For self signed certificates only use for development
```

File: app/views/docs/getting-started-for-android.phtml [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+<h2>Full Example</h2>
+<div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+                    <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import io.appwrite.Client
+import io.appwrite.services.Account
+
+val client = Client(context)
+  .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+  .setProject("5df5acd0d48c2") // Your project ID
+  .setKey("919c2d18fb5d4...a2ae413da83346ad2") // Your secret API key
+  .setSelfSigned() //remove in production
```
Comment (abnegate) [inline app/views/docs/getting-started-for-android.phtml:110]:
```suggestion
  .setSelfSigned(true) // For self signed certificates only use for development
```

**Pull request merged**